packages/webcert/src/feature/certificate/UnifiedEdit/UeInteger/UeInteger.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  CertificateDataElement,
+  ConfigUeInteger,
+  QuestionValidationTexts,
+  TextInput,
+  ValidationError,
+  ValueInteger,
+} from '@frontend/common'
+import { ValidationWrapper } from '@frontend/common/src/components/Inputs/DatePickerCustom/Styles'
+import * as React from 'react'
+import { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateCertificateDataElement, updateClientValidationError } from '../../../../store/certificate/certificateActions'
+import { getVisibleValidationErrors } from '../../../../store/certificate/certificateSelectors'
+
+const INVALID_NUMBER_PERIOD_ERROR = 'Ange ett värde mellan 0 och 100 %'
+
+export interface Props {
+  question: CertificateDataElement
+  disabled: boolean
+}
+
+const UeInteger: React.FC<Props> = ({ question, disabled }) => {
+  const dispatch = useDispatch()
+  const questionValue = question.value as ValueInteger
+  const questionConfig = question.config as ConfigUeInteger
+  const [number, setNumber] = useState<string>('')
+  const validationErrors = useSelector(getVisibleValidationErrors(question.id, 'NUMBER'))
+
+  const handleNumberOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const inputValue = event.target.value
+    const numberValue = Number(inputValue)
+    isNumberValid(numberValue)
+
+    setNumber(event.target.value.replace(/[^0-9]/g, ''))
```

Comment:
I think the component should probably be able to handle negative integers too, so regex should probably be something like /^-?[0-9]/g  (might be improved...)